### ISSUE TITLE
automatic color mode using latex css

### ISSUE THREAD

**Issue by 7zx** [vincentdoerig/latex-css#64]
How do i enable dark or light mode based on the user OS color preferences? I read the docs, but i didn't really understand, what do i have to change to enable this feature? I don't mean the toggle button by the way,

**Comment by vihuna** [comment]
It's explained in "Dark Mode" [section](https://latex.vercel.app/#dark-mode).

"If you want enable dark mode based on the prefers-color-scheme CSS media query, then use the latex-dark-auto class instead."

I.e. you only need to add the class `latex-dark-auto` to `body` HTML element: 

```html
<body id="top" class="latex-dark-auto">
```

**Comment by 7zx** [comment]
I did that, didn’t work

vihuna ***@***.***> escreveu em qua., 1/11/2023 às 18:52 :

**Comment by vihuna** [comment]
I tested it before my comment yesterday and I have tested again today (see video, Firefox on the left, Chrome on the right; Gnome DE).

https://github.com/vincentdoerig/latex-css/assets/60484272/ce7b89f9-fce0-4a5a-b06c-46f3344df55b

Because `prefers-color-scheme` involves Desktop/OS theme preferences, it can be more complicated; I don't know what happens in Windows, macOS.

It may be helpful if you give more information, like the OS/Desktop and the Browser that you are using.
